x-pack/plugins/actions/common/validate_email_addresses.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { parseAddressList } from 'email-addresses';
+import { ValidatedEmail, InvalidEmailReason } from './types';
+import { hasMustacheTemplate } from './mustache_template';
+
+export interface ValidateEmailAddressesOptions {
+  allowMustache?: boolean;
+}
+
+// this can be useful for cases where a plugin needs this function,
+// but the actions plugin may not be available.  This could be used
+// as a stub for the real implementation.
+export function validateEmailAddressesAsAlwaysValid(
+  addresses: string[],
+  options?: ValidateEmailAddressesOptions
```

Comment:
Nit: the `options` variable is unused, it looks like you can remove it

```suggestion
  addresses: string[]
```

Reply:
We may need it to match signatures.  We end up using this function instead of the "real" one, when we know we're always going to return all emails as valid.  So it has to match the "original", at least in some cases.  Worth a note, assuming I'm correct.  And maybe we can just remove it, and have signatures match "enough", but not sure ...